packages/web/webpack/client.ts

```diff
@@ -0,0 +1,90 @@
+import { join, resolve } from 'path';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import MiniCssExtractPlugin from 'mini-css-extract-plugin';
+import CopyPlugin from 'copy-webpack-plugin';
+import { configCommon, pluginsCommon, srcDir } from './common';
+import { Configuration } from 'webpack';
+
+const distDir = 'dist/static';
+
+export const clientConfig: (mode: 'development' | 'production', watch: boolean) => Configuration = (
+  mode,
+  watch
+) => ({
+  entry: join(__dirname, '..', srcDir, 'index.tsx'),
+  ...configCommon(mode),
+  module: {
+    rules: [
+      {
+        test: /\.(png|svg|jpg|jpeg|gif|woff(2)?|ttf|eot)$/i,
+        exclude: /sprite\/([^\/]*)\.svg$/,
+        type: 'asset/resource',
+      },
+      {
+        test: /sprite\/([^\/]*)\.svg$/,
+        loader: 'svg-sprite-loader',
+        options: {
+          extract: true,
+          spriteFilename: 'sprite.svg',
+        },
+      },
+      {
+        test: /\.(tsx|ts)?$/,
+        use: 'ts-loader',
+        exclude: '/node_modules/',
+      },
+      {
+        test: /\.module\.scss$/,
+        use: [
+          MiniCssExtractPlugin.loader,
+          {
+            loader: 'css-loader',
+            options: {
+              modules: true,
+              sourceMap: mode === 'development',
+            },
+          },
+          {
+            loader: 'sass-loader',
+            options: {
+              sourceMap: mode === 'development',
+            },
+          },
+        ],
+      },
+      {
+        test: /\.s?css$/,
+        exclude: /\.module.scss$/,
+        use: [
+          MiniCssExtractPlugin.loader,
+          'css-loader',
+          {
+            loader: 'sass-loader',
+            options: {
+              sourceMap: mode === 'development',
+            },
+          },
+        ],
+      },
+    ],
+  },
+  plugins: [
+    ...pluginsCommon(mode, false),
+    new HtmlWebpackPlugin({
+      template: join(__dirname, '..', 'index.html'),
+    }),
+    new CopyPlugin({
+      patterns: [
+        { from: 'robots.txt', to: 'robots.txt' },
+        { from: 'src/assets/sharing-image.png', to: 'sharing-image.png' },
```

Comment:
`to: 'assets/sharing-image'`

Reply:
We don't have a separate `assets` folder now inside `dist/static`.

Reply:
It should be `static/sharing-image.png` than
https://gradejs.com/static/sharing-image.png